main.js

```diff
@@ -5005,6 +5025,26 @@ function createRoomCallback(passAdd, passAdd2) {
 	joinRoom(session.roomid);
 
 }
+/**
+ * Handles click actions on the room selection buttons in #controlButtons
+ * @param {string} room - Room name to select/deselect for the next transfer call
+ */
+function handleRoomSelect(room) {
+    var elems = document.querySelectorAll(".btnArmTransferRoom");
+    [].forEach.call(elems, function(el) {
+        el.classList.remove("selected");
+    });
+    if (previousRoom == room) {

```

Comment:
It is recommended to use three equal signs instead of two to use strict equality, it is very well explained in the ES lint documentation: https://eslint.org/docs/2.0.0/rules/eqeqeq